shaders/profiles/slang/profiles.frag.slang

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2025, Sascha Willems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+[[vk::binding(1, 0)]]
```

Comment:
Being explicit with the binding is likely not needed.  This is one of the cool differences between SLang and HLSL.

Reply:
Yeah, as noted in the description I only added those shaders so people can actually test the PR. They're mostly carbon copies of the HLSL ones.